packages/node_modules/@ciscospark/widget-meet/src/container.js

```diff
@@ -70,10 +70,34 @@ export class MeetWidget extends Component {
     props.hangupCall(media.get(`call`));
   }
 
+  @autobind
+  handleStartSendingAudio() {
+    this.props.media.get(`call`).startSendingAudio();
```

Comment:
I have mixed feelings about doing these actions this way and not using redux actions. I guess since we have all of the media event listeners setup, it doesn't really matter.